chips/sam4l/src/usbc/mod.rs

```diff
@@ -493,7 +493,7 @@ impl<'a> Usbc<'a> {
     }
 
     /// Enable the controller's clocks and interrupt and transition to Idle state
-    fn _enable(&self, mode: Mode) {
+    fn enable(&self, mode: Mode) {
```

Comment:
Is there a reason why all of these methods had a leading underscore, or was this just written by a Python developer? :smile:

Reply:
For me, the real question is how did `_disable` evade this check?
https://github.com/tock/tock/pull/4193/files#diff-3aea2c215a84ac01554328f9ae3ff921518ace5ef279feb8801fe5fd65fd6684R557

Reply:
`_disable_ isn't used anywhere. That's the difference. My feeling is that it's worth keeping in as dead code for if/when eventually it is used since I'm nearly certain it's correct.